app/controllers/projects/samples_controller.rb

```diff
@@ -57,13 +57,12 @@ def update
     def destroy
       ::Samples::DestroyService.new(@sample, current_user).execute
       @pagy, @samples = pagy(Sample.where(project_id: @project.id))
-      respond_to do |format|
-        if @sample.deleted?
-          format.turbo_stream do
-            render status: :ok, locals: { type: 'success',
-                                          message: t('.success', sample_name: @sample.name) }
-          end
-        else
+
+      if @sample.deleted?
+        flash[:success] = t('.success', sample_name: @sample.name)
```

Comment:
Can I get you to update the success message in the translation file for this to indicate that the `Sample SAMPLE_NAME was successfully removed from project %{project_name}`

Reply:
Fixed in 6d601665b86ef86bed23e499d59e7850fac4dd25